### PR TITLE
New package: triangle-1.6

### PR DESCRIPTION
**Pull request by olafmersmann**
Add package triangle, a 2d mesh generation utility.

### CODE REVIEW

**Comment by biopsin** [comment]
Please submit each package as a seperate pull reguest.

**Comment by ericonr** [comment]
Please don't open and reopen PRs like this. Maintainers get emails for each of these PRs.

Use git rebase to remove the commits that don't belong to the branch as well.

File: srcpkgs/triangle/files/LICENSE [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Berkeley, California  94705-1927
+jrs@cs.berkeley.edu
+
+This program may be freely redistributed under the condition that the
+copyright notices (including this entire header and the copyright    
+notice printed when the `-h' switch is selected) are not removed, and 
+no compensation is received.  Private, research, and institutional   
+use is free.  You may distribute modified versions of this code UNDER
+THE CONDITION THAT THIS CODE AND ANY MODIFICATIONS MADE TO IT IN THE
+SAME FILE REMAIN UNDER COPYRIGHT OF THE ORIGINAL AUTHOR, BOTH SOURCE
```
Comment (Chocimier) [inline srcpkgs/triangle/files/LICENSE:13]: > MODIFICATIONS ... REMAIN UNDER COPYRIGHT OF THE ORIGINAL AUTHOR

If I understand this correctly, this is try to take over rights to modifications from author of modification. Unacceptable even as restricted.

Comment (olafmersmann) [reply srcpkgs/triangle/files/LICENSE:13]: So it should go into nonfree?

P.S.: Sorry for all the noise, still learning.

**Pull request closed without merging**